package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "doc": "./node_modules/.bin/jsdoc lib/api/1.1/northbound -r -d docs",
     "test": "./node_modules/.bin/mocha $(find spec -name '*-spec.js') -R spec --require spec/helper.js",
-    "install": "./install-web-ui.sh && ./install-swagger-ui.sh",
+    "install": "./install-web-ui.sh && ./install-swagger-ui.sh && ./install-task-doc.sh",
```

Comment:
Agreed, moved to taskdoc command and modified relative build scripts.
@WangWinson @yyscamper